flytekit/annotated/sample.py

```diff
@@ -0,0 +1,13 @@
+from flytekit.annotated.stuff import task, workflow, WorkflowOutputs
+
+
+@task(outputs=['s_out'])
+def x(s: int) -> int:
+    return s + 1
+
+
+@workflow(outputs=["real_b"])
+def my_workflow() -> WorkflowOutputs:
+    # a = x(s=3)
+    b = x(s=x(s=3))
```

Comment:
can we get a little more complex example, where we have 3 nodes, multiple outputs and inputs?